Assets/Scripts/DaggerfallUnityStructs.cs

```diff
@@ -221,6 +221,7 @@ public struct MobileEnemy
         public bool HasSpellAnimation;              // Whether or not this character has specific animations for casting spells
         public int[] SpellAnimFrames;               // Animation sequence to play when doing a spell cast
         public MobileTeams Team;                    // Team that this enemy uses if enemy in-fighting is on
+        public bool AlliedToPlayer;                 // Overrides standard target choices to defend player
     }
```

Comment:
We could use a bool, but I had planned to use the "teams" functionality I had wanted to add the summoned daedroth to the "player allies" team.